scalafix-tests/src/test/resources/checkSyntax/NoAutoTupling.source

```diff
@@ -0,0 +1,89 @@
+rewrites = [NoAutoTupling]
```

Comment:
I haven't updated the other test suites to take advantage of the fact that you can now skip the `[]` and write `rewrites = NoAutoTupling`. Gonna add support to write the singular `rewrite = NoAutoTupling`. I will update it in a future PR, so don't worry about it, just an FYI :)